src/DependencyInjection/LazyContainerFactory.php

```diff
@@ -345,9 +345,7 @@ final class LazyContainerFactory
         ParamTypeResolver::class,
         PropertyFetchTypeResolver::class,
         PropertyTypeResolver::class,
-        ReturnTypeResolver::class,
         ScalarTypeResolver::class,
-        StaticCallMethodCallTypeResolver::class,
```

Comment:
if this `StaticCallMethodCallTypeResolver` removed from the list, the `StaticCallMethodCallTypeResolver.php` should be removed as well.